bindata/assets/deployments/console-deployment.yaml

```diff
@@ -21,6 +21,7 @@ spec:
         component: ui
       annotations:
         target.workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
+        openshift.io/required-scc: restricted-v2
```

Comment:
sorry @liouk  May I know why it is pinned to `restricted-v2` instead of other default SCCs?

Reply:
The console pods are anyway getting admitted with `restricted-v2`, as they don't need more privileged access. Here we are just pinning the default SCC that the pods are already getting admitted with.